src/synapse/workload.py

```diff
@@ -517,7 +517,7 @@ def _get_irc_bridge_config(charm_state: CharmState, db_connect_string: str) -> t
     config["database"]["connectionString"] = db_connect_string
     if charm_state.synapse_config.irc_bridge_admins:
         config["ircService"]["permissions"] = {}
-        for admin in (a.strip() for a in charm_state.synapse_config.irc_bridge_admins.split(",")):
+        for admin in charm_state.synapse_config.irc_bridge_admins:
             config["ircService"]["permissions"][f"@{admin}"] = "admin"
```

Comment:
Wasn't the @ already added? (I mean in `[f"@{admin}"] `). Maybe a test could help...

Reply:
it is removed as juju interprets it as a pointer to a file, so we add it in the code and not via the action

Reply:
but I believe the `@` was already added in the function userids_to_list. I believe there could be two `@` symbols

Reply:
excellent catch. Fixed.